content/manager/bootstrapping.md

```diff
@@ -165,6 +165,97 @@ Services:
 {{< /gsHighlight >}}
 
 
+# Deploying a Manager Image
+
+{{% gsNote title="Prerequisites" %}}
+ * A manager image built using the [Image Builder](https://github.com/cloudify-cosmo/cloudify-image-bakery/tree/master/cloudify_marketplace)
+ * For the command-line install, the [`cfy` command]({{< relref "/cli/overview" >}})
```

Comment:
Which link should that be? /intro/installation is the best fit I could find so I've put that for now.